pymdp/distribution.py

```diff
@@ -90,8 +90,8 @@ def __setitem__(self, indices, value):
         ]
         self.data[tuple(index_list)] = value
 
-    def normalize(self):
-        self.data = norm_dist(self.data)
+    def normalize(self, add_noise: float = 0.0):
```

Comment:
Aside from my argument that `add_noise` shouldn't be part of this function, I also think it should be called `epsilon` or something since the term `noise` suggests there's some sampling happening under the hood.